signatures/LUOV/LUOV/bench/avx2-chacha8_LUOV-8-107-371_asus.yaml

```diff
@@ -0,0 +1,8 @@
+platform: ASUS S410 Notebook PC with an Intel Core i5-8250U CPU @ 1.60 GHz
```

Comment:
(@all files in `bench/`) I'd prefer if we named the files `kabylake` instead of `asus`, as that's the more relevant info.